agent/engine/task_manager.go

```diff
@@ -351,8 +351,6 @@ func (mtask *managedTask) waitEvent(stopWaiting <-chan struct{}) bool {
 		mtask.handleDesiredStatusChange(acsTransition.desiredStatus, acsTransition.seqnum)
 		return false
 	case dockerChange := <-mtask.dockerMessages:
-		seelog.Infof("Managed task [%s]: got container [%s (Runtime ID: %s)] event: [%s]",
```

Comment:
this is redundant because it gets logged immediately on entering the handleContainerChange function